util/cron/test-homebrew.bash

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+
+
+
+# Create a tarball from current repo. Beware, this will corrupt the state of
```

Comment:
What does this mean?  In what way will the current state of the repo be corrupted?

Reply:
Other scripts using ./util/buildRelease/gen_release ${version} to create tarball is having this comment so I added it(My best guess is the script is creating a bunch of dirs /tmp and /tar so a warning is added if the gen_release script is  executed in a local machine).
Since this will run in a test mac machine which will get a fresh clone of the repo overtime the test is run, I removed it.